### PR TITLE
Tweaks to copy as ESModule url context menu item

### PR DESCRIPTION
**Pull request by DeMoorJasper**
## What kind of change does this PR introduce?

Implements some final touches to the esmodule url context menu item.

- Only show the item for ts, tsx, js and jsx
- Use the new codesandbox hostname
- Add the new badge

Screen Shot 2021-07-12 at 14 20 42

## Checklist

- [x] Documentation
- [x] Testing 
- [x] Ready to be merged
      
- [x] Added myself to contributors table

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/5962/commits/f229de6f64f3d11cb6de1f924c0c90f797bb6336 is at https://pr5962.build.csb.dev/s/new.

**Review by danilowoz — APPROVED** [review]
The code looks good to me! But I'd wait for Danny's review before merging it.

**Pull request merged**
